easybuild/easyconfigs/b/Bazel/Bazel-0.29.1-GCCcore-8.3.0.eb

```diff
@@ -15,7 +15,10 @@ checksums = [
     '99928d0902beeaf962a8ad14db8432f8e5114645e3caf64c7ee2fa136c31609f',  # Bazel-0.29.1_fix-gold-flag.patch
 ]
 
-builddependencies = [('binutils', '2.32')]
+builddependencies = [
+  ('binutils', '2.32'),
+  ('Python', '3.7.2'),
```

Comment:
@Flamefire This should be Python 3.7.4 (resolved via `Python-3.7.4-GCCcore-8.3.0.eb`)

Reply:
Thanks!